content/enterprise/admin/infrastructure/demo-to-disk-migration.mdx

```diff
@@ -0,0 +1,147 @@
+---
+page_title: Demo to Mounted Disk Migration - Infrastructure Administration - Terraform Enterprise
+description: >-
+  Learn how to migrate Terraform Enterprise from _Demo_ mode to _Mounted Disk_
+  mode.
+---
+
+# Migrating from Demo Mode to Mounted Disk Mode
+
+Terraform Enterprise removed the _Demo_ operational mode in v202204-1. We
+strongly recommend upgrading your instance from _Demo_ mode to _Mounted Disk_
+mode.
+
+If you are currently running Demo mode and wish to upgrade to Terraform
+Enterprise v202204-1, you must migrate your application data from the _Demo_
+mode Docker volumes to the _Mounted Disk_ directory.
+
+## How do I check if I'm running demo mode?
+
+Connect to your Terraform Enterprise instance and run the following command.
+
+```sh
+replicatedctl app-config export --template '{{ .installation_type.Value }}'
+```
+
+If you see the value `poc` that means you are running demo mode and you'll need
+to migrate your application data to the mounted disk directory.
+
+## What happens if I upgrade to Terraform Enterprise v202204-1 without migrating my application data?
+
+Terraform Enterprise will fail to start with the following error until you
+migrate your application data.
+
+```
+Unmet start requirement: Required setting "Mounted Disk Path" is not set.
+```
+
+If you run `replicatedctl app status`, you will see output similar to the
+following.
+
+```json
+[
+    {
+        "AppID": "f9486004126d40904381e029d76353d0",
+        "Sequence": 609,
+        "PatchSequence": 0,
+        "State": "stopped",
+        "DesiredState": "started",
+        "Error": "Unmet start requirement: Required setting \"Mounted Disk Path\" is not set.",
+        "IsCancellable": false,
+        "IsTransitioning": false,
+        "LastModifiedAt": "2022-04-11T21:02:18.613686589Z"
+    }
+]
+```
+
+The Replicated console shows the following page.
+
+![A screenshot showing the error message on the Replicated console.](/img/docs/demo-to-disk.png)
+
+## Migrating Application Data From Demo Mode to Mounted Disk Mode
+
+1. Connect to your Terraform Enterprise instance.
+
+1. Create the Mounted Disk directory. You can choose any directory, but Terraform Enterprise expects that the directory is backed by persistent storage that can be mounted to the instance.
```

Comment:
This is the only place we capitalize mounted disk. I feel like....I can't tell whether it should be or not. Are we referring to the mode name, or is mounted disk just a THING. If it's not referring to the mode name, I say no caps.